esmvalcore/cmor/_fixes/fix.py

```diff
@@ -2,15 +2,30 @@
 import importlib
 import inspect
 import os
+from functools import lru_cache
+from pathlib import Path
+
+import yaml
+
+from esmvalcore._config._config import CFG
 
 from ..table import CMOR_TABLES
 
 
+@lru_cache
+def get_variable_mappings(project, dataset):
+    DEFAULT_PATH = (Path(__file__).parents[0] / project /
```

Comment:
project names and dataset names are usually used in lower case when looking in source directories